docs/customise-client/api.rst

```diff
@@ -62,17 +62,17 @@ For instance, here is the customisation used by nextstrain.org:
 +--------------------------+------------------------------+----------------------------------------------------+
 | Properties               | CSS string of                | Description                                        |
 +==========================+==============================+====================================================+
-| color                    | color                        | Text color                                         |
-+--------------------------+------------------------------+----------------------------------------------------+
 | selectedColor            | color                        | Text color of selected text / button text          |
 +--------------------------+------------------------------+----------------------------------------------------+
 | unselectedColor          | color                        | Text color of unselected text / button text        |
 +--------------------------+------------------------------+----------------------------------------------------+
+| color                    | color                        | Text color of all other text                      |
```

Comment:
This line needs an extra space (which is why CI is failing)

Reply:
Whoops, fixed in [force-push](https://github.com/nextstrain/auspice/compare/41f536bd834862237b5beb08df0dce872046c7e2..e354e374b98c870995a66c1afc7c836a9d5bcef1). Useful CI!